db/migrate/20231011103114_migrate_notification_to_noticed.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class MigrateNotificationToNoticed < ActiveRecord::Migration[7.0]
+  def up
+    add_column :notifications, :params, :jsonb, default: {}, null: false
+    add_column :notifications, :type, :string, null: false, default: 'LegacyNotification'
```

Comment:
Why LegacyNotification became default?

Reply:
don't use default, just set it below when migrating